src/components/ContactUsForm/index.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Col, Row } from "antd";
+import { RiSendPlaneFill } from "react-icons/ri";
+import "./index.css";
+
+const ContactUsForm = () => {
+  const [t] = useTranslation();
+  const [formValue, setFormValue] = useState({
+    name: "",
+    email: "",
+    message: "",
+  });
+
+  // The change in isSubmitted state triggers submission message
+  const [isSubmitted, setIsSubmitted] = useState(false);
+
+  // Converts object to proper form data(string)
+  const encode = (data) => {
+    return Object.keys(data)
+      .map(
+        (key) => encodeURIComponent(key) + "=" + encodeURIComponent(data[key])
+      )
+      .join("&");
+  };
+
+  const handleChange = (e) => setFormValue({ [e.target.name]: e.target.value });
+
+  // application/x-www-form-urlencoded stands for POST data at HTTP.
+  // body represents HTTP message and send data to server in a query string format.
+  // encode is used for passing the string to the body
+  // setTimeOut stands for showing submission message about a time after submit
+  const handleSubmit = (e) => {
+    fetch("/", {
+      method: "POST",
+      "data-netlify": "true",
+      headers: { "Content-Type": "application/x-www-form-urlencoded" },
+      body: encode({ "form-name": "contact", ...formValue }),
+    }).then(() => {
+      setTimeout(() => setIsSubmitted(false), 5000);
```

Comment:
this timeout is weird. you probably don't want to do this, and 5 seconds is a long time for the user to wait. you may want a state called isLoading or something (ibrahim did something similar) but in general we never write code with manual timeouts like this. it could take, in reality, longer than 5 seconds.

what's the reason?

Reply:
The main reason I have used timeout (via using isSubmitted state) is that showing post submit message a little more to the user. I gave 5 seconds by trying and it seems I underestimated back sides of it. I'm looking Ibrahim's code, I'll try adapting it to  handleSubmit function.